MATLAB.gitlab-ci.yml

```diff
@@ -111,17 +111,17 @@ test_artifacts:
 # [8] https://www.mathworks.com/help/matlab/ref/matlab.unittest.plugins-package.html
 # [9] https://www.mathworks.com/help/matlab/matlab_prog/generate-artifacts-using-matlab-unit-test-plugins.html
 
-# Starting in R2022b, the `build` job runs a build using the MATLAB build tool. When you use the `build` job, a file 
-# named `buildfile.m` must be in the root of your repository. You can use this job to run the tasks specified in 
-# `buildfile.m`. To use the job in your pipeline, substitute `mytask` with the tasks you want to run.
+# Starting in R2022b, the `build` job runs a build using the MATLAB build tool. You can use this job to run the
+# tasks specified in a file named `buildfile.m` in the root of your repository. 
 #
 build:
   extends: .matlab_defaults
-  script: matlab -batch "buildtool mytask"
+  script: matlab -batch "buildtool"
 
-# If you specify more than one task, use a space to separate them. If you do not specify any tasks, the job 
-# runs the default tasks in `buildfile.m` as well as all the tasks on which they depend. The job executes the 
-# [`buildtool`][10] command to run a build using the tasks and options that you specify.
+# The job runs the default tasks in `buildfile.m` as well as all the tasks on which they depend. If you wish to run a 
```

Comment:
Use this paragraph after the code instead:

The job executes the [`buildtool`][10] command to run a build using the default tasks in `buildfile.m` as well as all the tasks on which they depend. To run specific tasks instead, specify them as a space-separated list in the job. For example, to run the tasks named `task1` and `task2` and their dependencies, substitute `"buildtool"` with `"buildtool task1 task2"`.